makejdk-any-platform.1

```diff
@@ -166,8 +166,7 @@ For reference, OpenJDK version numbers look like 1.8.0_162-b12 (for Java 8) or
 This is typically used in conjunction with \fB<-b>\fR.
 .TP
 .BR \-\-use-jep319-certs
-Use certs defined in JEP319 in Java 8/9. This will increase the volume of traffic
-downloaded, however will provide an uptodate ca cert list.
+Use certs defined in JEP319 in Java 8/9. Deprecated, has no effect.
```

Comment:
Also need the custom certs flag added to this file

Reply:
Done